fallout3.asl

```diff
@@ -10,5 +18,5 @@ exit
 
 isLoading
 {
-    return current.isLoading;
+    return vars.isLoading;
```

Comment:
Instead of adding an update block and a var, you can just do `return current.isLoading || !current.introDone;` here

Reply:
Ah thank you :)